modules/internal/ChapelUtil.chpl

```diff
@@ -128,7 +128,7 @@ module ChapelUtil {
 
   // param s is used for error reporting
   pragma "command line setting"
-  proc _command_line_cast(param s: c_string, type t, x:c_string) {
+  proc _command_line_cast(param s: chpl_c_string, type t, x:c_ptrConst(c_char)) {
```

Comment:
Might be worth a comment that `param` `c_ptr(c_string)` does not exist, so this will have to happen in a different way once `chpl_c_string` is fully removed. Ideally it would use `param s: string`, but that will require some compiler modifications.

Reply:
added a comment explaining this